idranges.toml

```diff
@@ -51,3 +51,20 @@ ror_id = ""
 # [vocabs.othervocab.checks]
 # [vocabs.othervocab.prefix_map]
 # [[vocabs.othervocab.idrange]]
+
+[vocabs.biocatalysis_demo]
+id_length = 7
+permanent_iri_part = "https://w3id.org/nfdi4cat/voc4cat_biocatalysis/"
```

Comment:
Better use example.org as test domain so that w3id.org does not get the unresolvable requests.
...or (even better) the GitHub pages URL where the docu will land. Should be `https://nfdi4cat.github.io/voc4cat-playground/latest/biocatalysis_demo/index.html` for your vocabulary named `biocatalysis_demo`.

Reply:
OK, done :)